src/object/schema_manager.c

```diff
@@ -4446,6 +4446,10 @@ sm_update_all_catalog_statistics (bool with_fullscan)
     CT_METHFILE_NAME, CT_QUERYSPEC_NAME, CT_INDEX_NAME,
     CT_INDEXKEY_NAME, CT_CLASSAUTH_NAME, CT_DATATYPE_NAME,
     CT_COLLATION_NAME, CT_CHARSET_NAME, CT_SYNONYM_NAME,
+    CT_STORED_PROC_NAME, CT_STORED_PROC_ARGS_NAME, CT_PARTITION_NAME,
+    CT_SERIAL_NAME, CT_COLLATION_NAME, CT_USER_NAME,
+    CT_TRIGGER_NAME, CT_PASSWORD_NAME, CT_CHARSET_NAME,
+    CT_DB_SERVER_NAME, CT_SYNONYM_NAME,
```

Comment:
The following is redundant.
- CT_COLLATION_NAME
- CT_CHARSET_NAME
- CT_SYNONYM_NAME

And was it intentional to exclude the following?
- CT_AUTHORIZATION_NAME
- CT_HA_APPLY_INFO_NAME

Reply:
I fixed it.